app/webpack/stylesheets/interruption-panel.scss

```diff
@@ -2,8 +2,12 @@
   background-color:#1d70b8;
   padding:30px;
 
-  p,h1,h2,ul,label,a,a:link,a:visited {
```

Comment:
I don't think the changes to this file are needed. This is reverting changes that were made for the DWP Override functionalilty and are needed on master.